styleguide/source/assets/scss/03-organisms/_masthead.scss

```diff
@@ -40,6 +40,12 @@
 
     &__title {
       width: 100%;
+
+      &--centered {
+        p, h1, h2, h3, h4, h5, h6 {
```

Comment:
perhaps a wildcard would work better? *

Reply:
I don't think you can put a wildcard on just headers and I do not want to target everything inside --centered.